oneview/provider.go

```diff
@@ -89,6 +89,7 @@ func Provider() terraform.ResourceProvider {
 			"oneview_storage_attachment":         dataSourceStorageAttachment(),
 			"oneview_server_profile":             dataSourceServerProfile(),
 			"oneview_server_profile_template":    dataSourceServerProfileTemplate(),
+			"oneview_storage_pool":               dataSourceStoragePool(),
```

Comment:
I am not sure the reason for this change. Can you please explain.

Reply:
In this file we mention the datasource and resource file for each resources . The datasource for storage pool got removed sometime in past . I jsut added it back.